_sass/mixins/_rotate.scss

```diff
@@ -0,0 +1,6 @@
+@mixin rotate($degrees, $rotation) {
+  filter: progid:DXImageTransform.Microsoft.BasicImage(rotation=#{$rotation});
+  -webkit-transform: rotate($degrees);
+      -ms-transform: rotate($degrees);
```

Comment:
Line should be indented 2 spaces, but was indented 6 spaces<br>Avoid vendor prefixes.